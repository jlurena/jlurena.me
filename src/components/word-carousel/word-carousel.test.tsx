import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import WordCarousel from './word-carousel';

describe('<WordCarousel />', () => {
  it('renders correctly (snapshot)', () => {
    const words = ['Hello', 'World', 'Goodbye'];

    const { asFragment } = render(<WordCarousel words={words} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
