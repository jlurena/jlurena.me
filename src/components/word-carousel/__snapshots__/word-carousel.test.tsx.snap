// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<WordCarousel /> > renders correctly (snapshot) 1`] = `
<DocumentFragment>
  <span
    class="_word_6729c1"
  >
    Hello
  </span>
  <span
    class="_word_6729c1"
  >
    World
  </span>
  <span
    class="_word_6729c1"
  >
    Goodbye
  </span>
</DocumentFragment>
`;
